src/event_form/steps/event-form-step3.controller.js

```diff
@@ -245,7 +284,11 @@ function EventFormStep3Controller(
       _.each(locations, function(location, key) {
         locations[key] = jsonLDLangFilter(locations[key], language, true);
       });
-      var sortedLocations = locations.sort(orderLocationsByLevenshtein(filterValue));
+      // never show dummy locations in suggestion box
+      var locationsWithoutDummy = locations.filter(function(location) {
+        return !location.isDummyPlaceForEducationEvents;
+      });
```

Comment:
Why would they be included in the search results? @corneelwille said that the dummy locations would not have a valid address, so they should never be included in the possible suggestions anyway?

Reply:
Correct. But I used a location with a zipcode as a dummy location on my local environment, so it did appear in the suggestions. Do you want me to remove this piece?

Reply:
No I guess it's fine to leave it in just to be safe. 👍